cmd/agent/subcommands/diagnose/README.md

```diff
@@ -48,3 +49,6 @@ Normally a successful diagnosis is printed as a single dot character. If verbose
 
 ### ```local``` option
 Normally internal diagnose functions will run in the context of agent and other services. It can be overridden via –run-as-user options and if specified diagnose functions will be executed in context of the agent diagnose CLI process if possible.
```

Comment:
```suggestion
Normally, internal diagnose functions run in the context of the Agent and other services. It can be overridden with –run-as-user options, and if specified, diagnose functions are executed in the context of the Agent diagnose CLI process if possible.
```